src/bin/sccache-dist/main.rs

```diff
@@ -265,7 +265,12 @@ fn create_jwt_server_token(
     jwt::encode(header, &ServerJwt { server_id }, &key).map_err(Into::into)
 }
 fn dangerous_insecure_extract_jwt_server_token(server_token: &str) -> Option<ServerId> {
-    jwt::dangerous_insecure_decode::<ServerJwt>(server_token)
+    let mut validation = jwt::Validation::default();
+    validation.validate_exp = false;
+    validation.validate_nbf = false;
```

Comment:
This doesn't work
```
error[E0451]: field `validate_signature` of struct `Validation` is private
   --> src/bin/sccache-dist/main.rs:272:15
    |
272 |             ..jwt::Validation::default()
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^ field `validate_signature` is private
```

I did a slightly different approach to prevent the variable from being mutable after initialization